src/sparseml/pytorch/utils/sparsification_info/helpers.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List, Optional
+
+import torch
+from pydantic import BaseModel, Field
+
+
+__all__ = ["get_leaf_operations", "is_quantized"]
+
+
+def get_leaf_operations(model: torch.nn.Module) -> List[torch.nn.Module]:
+    """
+    Get the leaf operations in the model
+    (those that do not have operations as children)
+
+    :param model: the model to get the leaf operations from
+    :return: a list of the leaf operations
+    """
+    return [
+        submodule
+        for submodule in model.modules()
+        if len(list(submodule.children())) == 0
+    ]
+
+
+def is_quantized(operation: torch.nn.Module) -> bool:
+    """
+    Check whether the operation is quantized
+    """
+    if hasattr(operation, "qscheme"):
+        return operation.qscheme is not None
+    return False
+
+
+class QuantizationDtype(BaseModel):
```

Comment:
might as well just pass the quantization scheme directly here since it has the full information rather than creating a new scheme